src/message/MessageContainer.js

```diff
@@ -64,7 +64,7 @@ export default class MessageContainer extends React.PureComponent {
       pushRoute,
     } = this.props;
     const MessageComponent = isBrief ? MessageBrief : MessageFull;
-    const childrenNodes = htmlToDomTree(message.match_content || message.content);
+    const childrenNodes = htmlToDomTree((message.match_content || message.content).replace(/\r\n/g, '<br />').replace(/[\r\n]/g, '<br />'));
```

Comment:
@vishwesh3 we do ignore them any other place if it is not inside a `pre` tag. As in, this is the HTML spec. We had a lot of hacks before, that were one-offs and they definitely didn't work out. You can do the replacement if you want but must ensure it is only inside a `pre` tag.